doc/header.html

```diff
@@ -54,4 +54,27 @@
 </table>
 </div>
 <!--END TITLEAREA-->
+<script type="text/javascript">
+//<![CDATA[
+function getLabelName(innerHTML) {
+    var str = innerHTML;
+    var charCode;
+    for (var i = 0; i < str.length; i++) {
+        charCode = str.charCodeAt(i);
+        if (charCode > 96 && charCode < 123) {} else if (charCode > 64 && charCode < 91) {
+            str = str.substr(0, i) + String.fromCharCode(charCode + 32) + str.substr(i + 1);
+        } else if (charCode > 47 && charCode < 58) {} else if (charCode == 95) {} else if (charCode == 43) {
+            str = str.substr(0, i) + 'p' + str.substr(i + 1);
```

Comment:
no way you could replace some of those conditions with a .replace() outside of the for loop? It would be easier to understand.